fem/doftrans.hpp

```diff
@@ -15,19 +15,19 @@
 #include "../config/config.hpp"
 #include "../linalg/linalg.hpp"
 #include "intrules.hpp"
-#include "fe.hpp"
 
 namespace mfem
 {
 
-/** The DofTransformation class is an abstract base class for a family of
-    transformations that map local degrees of freedom (DoFs), contained within
-    individual elements, to global degrees of freedom, stored within
+/** The StatelessDofTransformation class is an abstract base class for a family
```

Comment:
Maybe describe what the term stateless means here.

Reply:
Good call. I have expanded this description.